src/auth.py

```diff
@@ -25,6 +25,25 @@
     "refresh_tokens",
 ]
 
+AUTH_DOMAIN = {
+    "prod": "https://formicbio.us.auth0.com",
+    "staging": "https://formicbio.us.auth0.com",
+    "edge": "https://edge.login.deeporigin.io",
+}
+
+AUTH_DEVICE_CODE_ENDPOINT = "/oauth/device/code"
+AUTH_TOKEN_ENDPOINT = "/oauth/token"
+AUTH_AUDIENCE = "https://os.deeporigin.io/api"
+
+
+AUTH_GRANT_TYPE = "urn:ietf:params:oauth:grant-type:device_code"
+
+AUTH_CLIENT_ID = {
```

Comment:
I see some drawbacks to this, in terms of security (although these are not keys they are half of the story) and maintainability. Is there any way devops could inject these -- and is it worth the effort?